FWCore/Utilities/interface/HRRealTime.h

```diff
@@ -76,6 +76,8 @@ namespace edm {
       __asm__ __volatile__("isb; mrs %0, cntvct_el0" : "=r"(ret));
       return ret;
     }
+#elif defined(__riscv)
+    static __inline__ unsigned long long rdtsc(void) { return 0; }
```

Comment:
I'd like to have a comment here (or a `#warning` as for ARMv7 above) explaining why this function returns 0 for RISC-V.

(in any case this support does not seem super important in the near or medium term, given that it seems to be used only in handful of unit tests)

Reply:
@makortel , my bad, I should not have included it here. Looking at https://gist.github.com/XieJiSS/18acd12685ecc0913d43d6bcfb3baefd , I see that there is way to get this info for riscv. You think something like the following should be enough  for riscv ?

```
#elif defined(__riscv) && __riscv_xlen == 64
static __inline__ unsigned long long rdtsc(void)
  uint64_t cycles;
  asm volatile("rdcycle %0" : "=r"(cycles));
  return cycles;
}
```

Reply:
> You think something like the following should be enough for riscv ?
> 
> ```
> #elif defined(__riscv) && __riscv_xlen == 64
> static __inline__ unsigned long long rdtsc(void)
>   uint64_t cycles;
>   asm volatile("rdcycle %0" : "=r"(cycles));
>   return cycles;
> }
> ```

I'm by far not an expert of RISC-V assembly, but based on quick searches it looks a reasonable starting point.

Reply:
done